README.md

```diff
@@ -79,9 +79,10 @@ pressing `C-c C-r`.
   [`no-littering`][no-littering].
 
 * `prescient-filter-method`: Which algorithm to use for filtering
-  candidates. The default is `literal+initialism` as described above,
-  but you can also use substring matching, initialism matching, regexp
-  matching, or fuzzy matching. See the docstring for full details.
+  candidates. The default is `literal` and `initialism` as described
+  above, but you can also use substring matching, initialism matching,
+  regexp matching, fuzzy matching, or any combination of those. See
+  the docstring for full details.
```

Comment:
I think it would be a bit clearer to start with "List of algorithms" rather than "Which algorithm".